core/primitives/src/version.rs

```diff
@@ -86,7 +86,6 @@ pub fn is_implicit_account_creation_enabled(protocol_version: ProtocolVersion) -
 #[derive(Hash, PartialEq, Eq, Clone, Copy, Debug)]
 pub enum ProtocolFeature {
     // stable features
-    ForwardChunkParts,
```

Comment:
Yes you are totally right. I updated the commit message to include the change. You are totally right that it's probably better to do it in a separate commit. I did it here because it was such a small change.